test/helpers/kubectl.go

```diff
@@ -4477,9 +4476,16 @@ func (kub *Kubectl) ensureKubectlVersion() error {
 		return err
 	}
 	path := path.Join(GetKubectlPath(), "kubectl")
+	rcVersion := fmt.Sprintf("v%s.0-rc.0", GetCurrentK8SEnv())
+	switch GetCurrentK8SEnv() {
+	// These versions never released a ".0". Only since 1.19 Kubernetes started
+	// to release RC starting from '0'.
+	case "1.16", "1.17", "1.18":
+		rcVersion = fmt.Sprintf("v%s.0-rc.1", GetCurrentK8SEnv())
```

Comment:
Why do we need the RC version for these ones if they already had a final release?

Reply:
(functionally I assume it's equivalent so if this is the easiest way then :+1: )

Reply:
Because I didn't think this through :facepalm: